actions/image/crop.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Image crop action
+ */
+
+$guid = get_input('guid');
+$entity = get_entity($guid);
+
+if (!$entity || !($entity instanceof ElggEntity) || !$entity->canEdit()) {
+	register_error(elgg_echo('image:crop:fail'));
+	forward(REFERER);
+}
+
+$x1 = (int) get_input('x1', 0);
+$y1 = (int) get_input('y1', 0);
+$x2 = (int) get_input('x2', 0);
+$y2 = (int) get_input('y2', 0);
+
+$filehandler = new ElggFile();
+$filehandler->owner_guid = $entity->getGUID();
+$filehandler->setFilename("icon/master.jpg");
```

Comment:
Perhaps we could later get also the image type as a parameter. So then we could have e.g:

- /dataroot/1/345/**icon**/size.jpg
- /dataroot/1/345/**cover_image**/size.jpg